src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -252,8 +271,9 @@ TEST_F(ClusterTaskManagerTest, ResourceTakenWhileResolving) {
   };
 
   /* Blocked on dependencies */
-  dependency_manager_.task_ready_ = false;
-  auto task = CreateTask({{ray::kCPU_ResourceLabel, 5}}, 1);
+  auto task = CreateTask({{ray::kCPU_ResourceLabel, 5}}, 2);
```

Comment:
Unrelated to this line of code, but maybe we can add a simple test that all args are pinned correctly?